src/teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'Person'
+
+class Teacher < Person
+  def initialize(name, age, specialization)
```

Comment:
- [ ]  Hello! 📝✨ It's important to note that the Student class inherits from the Person class, which means it should inherit all the parameters, including parent_permission with a default value of true. Additionally, the name parameter should have a default value of 'unknown'. 🚀📚 This ensures that our Student class inherits the necessary attributes from its parent class while also providing meaningful defaults where needed. 🧬👨‍👧‍👦📖